src/pages/lib/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -1,11 +1,456 @@
+
+<Accordion title='Review standard attributes options' headingLevel='4' eyebrow='Learn more'>
+
+There are many user attributes available to use by default in Cognito. Their definitions are based on the [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims):
+
+- `address`
+- `birthdate`
+- `email`
+- `family_name`
+- `gender`
+- `given_name`
+- `locale`
+- `middle_name`
+- `name`
+- `nickname`
+- `phone_number`
+- `picture`
+- `preferred_username`
+- `profile`
+- `zoneinfo`
+- `updated_at`
+- `website`
+
+You can find a [list of all the standard attributes here](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#cognito-user-pools-standard-attributes).
+
+You also have the ability to define custom attributes and up to 50 custom attributes can be added to your user pool. You can review the documentation on [defining custom attributes](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#user-pool-settings-custom-attributes) to learn more.
+
+</Accordion>
+
+### Pass user attributes during sign-up
+
+You can create user attributes during or after sign-up. To do this as part of sign-up you can pass them in the `attributes` object of `Auth.signUp` function parameters:
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify'
+
+async function handleSignUp() {
+  try {
+    await Auth.signUp({
+      username: 'jdoe',
+      password: 'mysecurerandompassword#123',
+      attributes: {
+        email: 'me@domain.com',
+        phone_number: '+12128601234', // E.164 number convention
+        given_name: 'Jane',
+        family_name: 'Doe',
+        nickname: 'Jane',
+        'custom:attribute_name': 'attribute_value'
+      }
+    });
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify'
+
+async function handleSignUp() {
+  try {
+    await Auth.signUp({
+      username: 'jdoe',
+      password: 'mysecurerandompassword#123',
+      attributes: {
+        email: 'me@domain.com',
+        phone_number: '+12128601234', // E.164 number convention
+        given_name: 'Jane',
+        family_name: 'Doe',
+        nickname: 'Jane',
+        'custom:attribute_name': 'attribute_value'
+      }
+    });
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+</Block>
+
+</BlockSwitcher>
+
+### Retrieve user attributes
+
+You can then retrieve user attributes for your users to read in their profile. This helps you personalize their frontend experience as well as control what they will see. Visible attributes will be limited to those you enabled during the `Specify read attributes` configured with the `auth` category. Then you can use the `currentAuthenticatedUser` method of the `Auth` class to read user attributes:
```

Comment:
the `auth` category - should that be a code highlight? Shouldn't it be "Auth"?

Reply:
Then you can use the `currentAuthenticatedUser` method to retrieve user attributes.